app/controllers/cloud_subnet_controller.rb

```diff
@@ -221,6 +222,7 @@ def process_cloud_subnets(subnets, operation)
     return if subnets.empty?
 
     if operation == "destroy"
+      deleted_subnets = 0
```

Comment:
`subnets.size` is number of subnets I want to delete and `deleted_subnets` is number of subnets that were deleted.  If only one subnet fails to be deleted than `subnets.size > deleted_subnets`.